lib/mappings/mappings.js

```diff
@@ -21,13 +21,41 @@ const amendMappingsBasedOnNyplSource = (data, nyplSource) => {
   }, {})
 }
 
+/**
```

Comment:
is MarcQuery a standard? Or is that just a name you gave this data structure?

Reply:
Not a standard. Just giving a name and strict structure to the plainobjects that `varFieldsMulti` expects.